.changeset/wet-hoops-drop.md

```diff
@@ -0,0 +1,17 @@
+---
+"fluid-framework": minor
+"@fluidframework/tree": minor
+---
+
+nodeChanged event now includes the list of properties that changed
+
+The payload of the `nodeChanged` event emitted by SharedTree now includes a `changedProperties` property that indicates
+which properties of the node changed.
+
+For object nodes, the list of properties uses the property identifiers defined in the schema, and not the persisted
+identifiers (or "stored keys") that can be provided through `FieldProps` when defining a schema.
+See the documentation for `FieldProps` for more details about the distinction between "view keys" and "stored keys".
```

Comment:
Should we use "property keys" here instead of "view keys"? I thought we preferred the former terminology.